pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -1002,12 +1002,25 @@ def make_backup(self):
             logging.warn(f'{backup_dir} already exists')
             self.set_state(TestState.ERROR)
             return
+        # Make a RockDB snapshot of the db to save space. This takes advantage of the immutability of sst files.
         logging.info(f'copying data dir to {backup_dir}')
-        shutil.copytree(self.target_near_home_path('data'),
-                        backup_dir,
-                        dirs_exist_ok=True)
-        logging.info(f'copied data dir to {backup_dir}')
-
+        cmd = [
+            self.data['current_neard_path'], '--home',
+            self.target_near_home_path(),
+            '--unsafe-fast-startup',
+            'database',
+            'make-snapshot',
+            '--destination',
+            backup_dir
+        ]
+        self.run_neard(cmd)
+        running = True
+        while running:
```

Comment:
oh i think this `while running` is kind of problematic no? It's quite sad to spin using nearly 100% cpu just to find out when neard exited. You could just try the `wait()` function of `self.neard`, since the whole `NeardRunner::poll_neard()` thing is not really needed for these one-time commands. And with that in mind, one other thing you could do is just to run this command with `subprocess.check_call()`. for example, look at how we run `neard init` in `NeardRunner::neard_init()`. It's just a lot simpler than using `run_neard()`, which was meant to run a longer running process that should continue to run after `neard_runner.py` exits